src/custom/pages/Faq/index.tsx

```diff
@@ -307,6 +320,26 @@ export default function Faq() {
             across different protocols. Since orders only incur a cost if traded, active market makers can observe the
             orderbook and place counter orders (creating a CoW) to prevent settling trades via external liquidity.
           </p>
+
+          <h3 id="what-are-gnosis-protocol-v2-solvers">What are Gnosis Protocol v2 Solvers?</h3>
+
+          <p>
+            In GPv2, instead of using a central operator or a constant function market maker to determine trade
```

Comment:
I also find this misleading. It may be worth adding the current plans (very well articulated in the text here) as well as a link to any related forum articles and maybe even a "Get involved deciding the future of solver competition on CowSwap!" to try and get some community participation in DAO decisions.

Reply:
Relevant conversation: https://gnosisinc.slack.com/archives/CNG8K061W/p1621415794004600

Reply:
I like your idea on encouraging people to encourage people to get involved